css/src/components/form/help.scss

```diff
@@ -0,0 +1,23 @@
+$help-color: $text !default;
+$help-error-color: $danger !default;
+$help-warning-color: $warning-active !default;
```

Comment:
`warning-dark` will have to do here, don't think we should pair it with a different concept

Reply:
updated